README.md

```diff
@@ -24,12 +28,43 @@ $ newman run https://www.getpostman.com/collections/631643-f695cab7-6878-eb55-79
 Custom templates (currently handlebars only) can be passed to the HTML reporter via `--reporter-html-template <path>` with `--reporters html` and `--reporter-html-export`.
 The [default template](https://github.com/postmanlabs/newman-reporter-html/blob/develop/lib/template-default.hbs) is used in all other cases.
 
+#### With Newman programmatically as a Node.js Library
+The CLI functionality is available for programmatic use as well.
+
+```javascript
+var newman = require('newman');
+
+newman.run({
+    collection: require('./sample-collection.json'),
```

Comment:
Add this file in `examples` if not already present, add a comment explaining that a file path can also be provided.